test/02LessonTwoTest.js

```diff
@@ -7,12 +7,25 @@ function whichSpecies(character) {
     // should return "cat" when character is 'garfield'
     // should return "fish" when character is 'nemo'
     // should return false if character is anything else
-
+    switch (character) {
+      case 'scooby':
+        return 'dog';
+      case 'garfield':
+        return 'cat';
+      case 'nemo':
+        return 'fish';
+      default:
+        return false;
+    }
 }
 
 function isEven(number) {
     // should return true is number is even (divisible by 2)
-
+    if (number % 2 === 0) {
+      return true;
+    } else {
+      return false;
+    }
 }
```

Comment:
Great job, @kieranypyne!

This would also work, fyi:

```javascript
function isEven(number) {
    // should return true is number is even (divisible by 2)
    return (number % 2 === 0);
}
```